htmlelements-java/pom.xml

```diff
@@ -29,6 +29,11 @@
             <version>2.6</version>
         </dependency>
         <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
We received this dependency transitively from older selenium-java. Now we should use it directly because it used in some places